### PR TITLE
Adding cache version bump hook

### LINKED ISSUE #84: Add a hook to bump the cache
We already have `do_action( 'minit-cache-purge-delete' )` for clearing the cache files. Let's add a hook for bumping the global cache version too.

```
do_action( 'minit-cache-version-bump' );
```

A follow-up to #83.

### PR DESCRIPTION
**Pull request by ryanhellyer**
Fixes #84

Adds extra hook for allowing plugins to bump the cache version number.

```
do_action( 'minit-cache-version-bump' );
```

### CODE REVIEW

**Comment by ryanhellyer** [comment]
I'm showing my lack of knowledge of how pull requests work here, as these were supposed to be all separate pull requests. I guess I need to use different branches or something to do that.

Anyhows, this tends to #83 and #82 as well now.

**Comment by szepeviktor** [comment]
@ryanhellyer There is a hack:

Locally remove 3 commits: `git reset HEAD~3` then force-push to the repo: `git push -f`

It is not a nice thing...

**Comment by kasparsd** [comment]
@ryanhellyer That's alright. Normally you would want to create one branch/pull request per issue to avoid any of them being a blocker for the rest of the features/fixes.

File: minit.php [hunk 1]
```diff
@@ -40,6 +40,9 @@ protected function __construct() {
 		// This action can used to delete all Minit cache files from cron
 		add_action( 'minit-cache-purge-delete', array( $this, 'cache_delete' ) );
 
+		// This action can be used to bump the cache version*
```
Comment (kasparsd) [inline minit.php]: @ryanhellyer Is the `*` at the end of the comment intentional?

File: readme.md [hunk 2]
```diff
@@ -12,8 +12,11 @@ Combine CSS files and Javascript files into single file in the correct order. Us
 
 ## Description
 
-TODO
+The cache is updated when the wp_enqueue version number changes or when the `Minit_Plugin:cache_bump()` method is fired.
 
+The complete cache can be purged by firing the `Minit_Plugin:cache_bump()` method.
```
Comment (kasparsd) [inline readme.md]: @ryanhellyer Let's suggest they use `do_action( 'minit-cache-version-bump' )` instead. That will avoid any issues if the plugin is de-activated or removed.

**Pull request closed without merging**